ravenframework/Optimizers/RavenSampled.py

```diff
@@ -301,7 +301,22 @@ def localFinalizeActualSampling(self, jobObject, model, myInput):
     # # testing suggests no big deal on smaller problem
     # the sign of the objective function is flipped in case we do maximization
     # so get the correct-signed value into the realization
-    if self._minMax == 'max':
+
+    self._objMult = {} #max will be -1, min will be 1
+    if not self._isMultiObjective:
+      if 'max' in self._minMax:
+        rlz[self._objectiveVar[0]] *= -1
+        self._objMult[self._objectiveVar[0]] = -1
+      else:
+        self._objMult[self._objectiveVar[0]] = 1
+    elif type(self._objectiveVar) == list:
+      for i in range(len(self._objectiveVar)):
```

Comment:
@joshua-cogliati-inl isn't this just `else` because `type(self._objectiveVar)` is always a list, even for single objective. Which means that the last else in line 319 will never be encountered.

Reply:
Hm. I can run all the tests without encountering any self._objectiveVar's that are not lists in RavenSampled, but it can be a single value in Optimizer (which RavenSampled inherits from). That said, there are other places in RavenSampled that would error out if _objectiveVar is not a list, so I think it can be switched to an else.

Reply:
I am running the full tests on this at: https://github.com/idaholab/raven/pull/2449

Reply:
Full tests passed on linux, so merged with NSGAII branch.

Reply:
Thanks a lot @joshua-cogliati-inl. @wangcj05 once this is all green, this will be ready for your review. Thank you both.